styles/_article.scss

```diff
@@ -86,12 +86,39 @@
     > p:last-child { margin-bottom: 0; }
   }
 
+  &-anchorbox {
+    background-color: #e8e8e8;
+    padding: 15px;
+    font-weight: $font-weight-bold;
+    margin-bottom: 30px;
+
+    & > p {
+      margin-top: 0px;
+      margin-bottom: 10px;
+    }
+
+    & > ul {
+      margin-bottom: 0px;
+      margin-top: 0px;
+      margin-left: 5px;
+    }
+  }
+
   &-content {
     line-height: 1.6;
     margin: 20px 0;
     word-wrap: break-word;
   }
 
+  img {
+    box-shadow: 0 2px 15px 0 rgba(0,0,0,.3);
+  }
+
+//hiding any uploaded files that are added to article from being displayed
+  &-attachments {
```

Comment:
Should we remove this from the template instead?

Reply:
yes I did that